build_luajit.sh

```diff
@@ -21,6 +20,13 @@ else
     if [ ! -d "raptorjit_repo" ]; then 
       git clone https://github.com/raptorjit/raptorjit raptorjit_repo
     fi
+
+    if [ -f "commitlock.txt" ]; then 
+      source commitlock.txt
+      echo "Current luajit commit hash: ${luajit_commitid}"
+      echo "Current raptorjit commit hash: ${raptorjit_commitid}"
+      (cd luajit_repo && git checkout ${luajit_commitid})
```

Comment:
The `(...)` brackets aren't necessary.

Reply:
I have to change the directory back if i don't run it like that

Reply:
Oh, interesting -- I didn't realise it had that effect!

Reply:
today I learned!